### PR TITLE
Add static icon for unknown user

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
Fixes https://github.com/nextcloud/mail/issues/9099

Add static icon for unknown user to prevent new avatar generation for each input.

**Before:**

https://github.com/nextcloud/mail/assets/1374172/f12d01dd-a252-4765-8e3e-eca95c125553

**After:** 

Screenshot from 2024-04-18 11-00-00

### CODE REVIEW

**Review by hamza221 — APPROVED** [review]

File: src/components/Composer.vue [hunk 1]
```diff
@@ -69,6 +69,7 @@
 							<ListItemIcon :no-margin="true"
 								:name="option.label"
 								:subtitle="option.email"
+								:iconClass="!option.id ? 'icon-user' : null"
```
Comment (ChristophWurst) [inline src/components/Composer.vue:72]: I am not sure if icon classes are still supported. I remember that we wanted to get rid of them an replace with inline (material) icons.

icon-class is not documented at https://nextcloud-vue-components.netlify.app/#/Components/NcListItems?id=nclistitemicon either

Comment (JuliaKirschenheuter) [reply src/components/Composer.vue:72]: > icon-class is not documented at 

`iconClass` is a valid prop of `NcAvatar` https://nextcloud-vue-components.netlify.app/#/Components/NcAvatar and will be thrown via attrs from `NcListItems`.

In current solution of `NcListItems` it is not possible to use material icons instead. Vue component need to be adjusted + NcVue release is needed.

**Review by ChristophWurst — APPROVED** [review]

File: src/components/Composer.vue (hunk 1 above)
Comment (ChristophWurst) [inline src/components/Composer.vue:72]: Attribute ':iconClass' must be hyphenated

**Review by st3iny — APPROVED** [review]
Tested and works.

**Comment by JuliaKirschenheuter** [comment]
CI is not related

**Pull request merged**